src/ccover/ccover_bmc.h

```diff
@@ -0,0 +1,107 @@
+/*******************************************************************\
+
+Module: Bounded Model Checking for Test Suite Generation
+
+Author: Daniel Kroening, kroening@kroening.com
+
+\*******************************************************************/
+
+/// \file
+/// Bounded Model Checking for Test Suite Generation
+
+#ifndef CPROVER_CCOVER_CCOVER_BMC_H
+#define CPROVER_CCOVER_CCOVER_BMC_H
+
+#include <list>
+#include <map>
+
+#include <util/invariant.h>
+#include <util/options.h>
+#include <util/ui_message.h>
+#include <util/decision_procedure.h>
+
+#include <goto-programs/goto_trace.h>
+
+#include <goto-symex/symex_target_equation.h>
+#include <goto-programs/goto_model.h>
+#include <goto-programs/safety_checker.h>
+#include <goto-symex/memory_model.h>
```

Comment:
I'm not sure all of the above are needed.

Reply:
done